ticker-sample/src/main/java/com/robinhood/ticker/sample/BaseActivity.java

```diff
@@ -15,7 +15,7 @@
     protected void onResume() {
         super.onResume();
         resumed = true;
-        handler.post(createRunnable());
+        handler.postDelayed(createRunnable(), 1000);
```

Comment:
👀 

just a sample, so nbd. why do you need the initial delay, though?

Reply:
its so i can verify the initial animation from default text to a new text (and ensure the animation works).